src/utils/i18n.js

```diff
@@ -208,6 +208,7 @@ i18n
           instructionsPanel: {
             emptyState: {
               addInstructions: "Add instructions",
+              removeInstructions: "Remove instructions",
```

Comment:
My only question is whether this should sit outside of the emptyState layer as it's in the view where there are instructions?